src/internal/components/InternalEditor.tsx

```diff
@@ -6,19 +6,19 @@ import {
   type History,
   InitialHistory,
 } from "@measured/puck";
-import React from "react";
+import React, { useEffect } from "react";
 import { customHeader } from "../puck/components/Header.tsx";
 import { useState, useRef, useCallback } from "react";
 import { getLocalStorageKey } from "../utils/localStorageHelper.ts";
 import { TemplateMetadata } from "../types/templateMetadata.ts";
 import { EntityFieldProvider } from "../../components/EntityField.tsx";
 import { SaveState } from "../types/saveState.ts";
 import { DevLogger } from "../../utils/devLogger.ts";
+import ThemeSidebar from "../puck/components/ThemeSidebar.tsx";
 
 interface InternalEditorProps {
   puckConfig: Config;
   puckInitialHistory: InitialHistory | undefined;
-  isLoading: boolean;
```

Comment:
As far as I could tell, `isLoading`, `canEdit`, and `change()` weren't doing anything -- `canEdit` was only read when updating `canEdit`